.github/workflows/tag-and-release.yaml

```
@@ -27,10 +27,14 @@ jobs:
     needs: tag-new-version
     if: ${{ needs.tag-new-version.outputs.release_created == 'true' }}
     runs-on: ubuntu-latest
```

```suggestion
    # TODO: (@WSTARR) since we don't run the tests on arm currently we don't need to use the 8 core
    runs-on: ${{ matrix.architecture == 'arm64' && 'uds-ubuntu-arm64-4-core' || 'uds-ubuntu-big-boy-8-core' }}
```

This is where things run